src/application/index.ts

```diff
@@ -3,21 +3,15 @@ import { createClient } from '../util/grpc'
 
 const defaultEndpoint = process.env.MESG_ENDPOINT
 
-var defaultApplication: Application
-
 type Options = {
   endpoint?: string
 }
 
 const applicationBuilder = (options?: Options) => {
```

Comment:
```suggestion
const applicationBuilder = (options?: Options): Application => {
```

No need to explicity set `Application` type as return of this function?

Reply:
Yeah, it's the best if we can declare all the types as much as we can to make type safety stronger. Though it's not 100% mandatory. Because, for example, in this case, we return the Application type. So, TS will already know the return type of this func. Thus, autocompletion and static analysis will work as expected.